accepted/2024/[TS]PMUI-Readme-rendering.md

```diff
@@ -0,0 +1,213 @@
+This is to ensure that users using the Browse tab will not see the README tab if they do not have a remote source that allows README downloads.
+
+If the README tab is rendered and we do not find a README then a message is rendered encouraging the user to contact the package author to upload a README.
+![alt text](../../meta/resources/ReadMePMUI/NoReadMeFound.png)
+
+##### Package Details
+This tab is always rendered and contains the package details information along with the Vulnerabilty and Depreciation information. 
+
+##### README File Sources
+
+* ReadmeFileUrl in the [package metadata](https://learn.microsoft.com/en-us/nuget/api/registration-base-url-resource).
+* README direct download specified in a new resource `ReadmeUriTemplate`.
+* Downloaded nupkg.
+
+#### NuGet API
+##### New RegistrationsBaseUrl version
+
+A new version of the [package metadata](https://learn.microsoft.com/en-us/nuget/api/registration-base-url-resource) resource type will be documented which will include the field **ReadmeFileUrl**.
+This will be a link to download the README and will only be filled if a readme is available to download.
+
+###### index.json
+
+```json
+{
+    "version": "3.0.0",
+    "resources": [
+        ...,
+        {
+            "@id": "https://apidev.nugettest.org/v3/registration5-gz-semver2/",
+            "@type": "RegistrationsBaseUrl/6.13.0"
+        },
+        ...
+    ]
+}
+```
+###### Example Response
+```json
+{
+    "@id": "https://apidev.nugettest.org/v3/registration5-gz-semver2/newtonsoft.json/index.json",
+    "items": [
+        {
+            ...,
+            "items": [ 
+                ...,
+                {
+                    "@id": "https://apidev.nugettest.org/v3/registration5-gz-semver2/newtonsoft.json/13.0.3.json",
+                    "@type": "Package",
+                    "catalogEntry": {
+                        "@id": "https://apidev.nugettest.org/v3/catalog0/data/2024.08.19.16.55.59/newtonsoft.json.13.0.3.json",
+                        "@type": "PackageDetails",
+                        "authors": "James Newton-King",
+                        "dependencyGroups": [ ... ],
+                        "description": "Json.NET is a popular high-performance JSON framework for .NET",
+                        "iconUrl": "https://apidev.nugettest.org/v3-flatcontainer/newtonsoft.json/13.0.3/icon",
+                        "id": "Newtonsoft.Json",
+                        "language": "",
+                        "licenseExpression": "MIT",
+                        "licenseUrl": "https://dev.nugettest.org/packages/Newtonsoft.Json/13.0.3/license",
+                        "readmeUrl": "https://dev.nugettest.org/packages/Newtonsoft.Json/13.0.3#show-readme-container",
+                        //New field
+                        "readmeFileUrl": "https://apidev.nugettest.org/v3/flatcontainer/newtonsoft.json/13.0.3/readme",
```

Comment:
This new field is described/introduced later in the document than this example. I think that's confusing, as someone who has been reading this document top to bottom.

Reply:
I moved it around a bit, hopefully its less confusing? Let me know.